node/src/parachain/peaq_chain_spec.rs

```diff
@@ -171,7 +171,7 @@ fn configure_genesis(
 			block_issue_reward: 7_909_867 * MILLICENTS,
 			max_currency_supply: 4_200_000_000 * DOLLARS,
 		},
-		vesting:peaq_runtime::VestingConfig { vesting: vec![] },
+		vesting: peaq_runtime::VestingConfig { vesting: vec![] },
```

Comment:
In the code above, we use the `Default::default()`; should we use the same way?

Reply:
It looks like Default::default() is the convention for leaving the configuration empty.